<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SerialPortAssistant</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QTextBrowser" name="textBrowser_rx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>350</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_tx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_fresh_com">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>FRESH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_onoff_com">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OPEN COM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_led">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hex_rx">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>HEX_RX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_str_rx">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>STR_RX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_rx">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hex_tx">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>HEX_TX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_str_tx">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>STR_TX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tx">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear_tx">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_com">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>50</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COM</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_baud">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_info1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rx_num">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tx_num">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_info0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
